iphone/cli/commands/_build.js

```diff
@@ -3167,7 +3182,8 @@ iOSBuilder.prototype.createXcodeProject = function createXcodeProject(next) {
 		};
 		xobjs.PBXShellScriptBuildPhase[buildPhaseUuid + '_comment'] = '"' + name + '"';
 	} else if (this.target === 'device') {
-		buildSettings.CODE_SIGN_IDENTITY = '"iPhone Developer: ' + this.certDeveloperName + '"';
```

Comment:
Doesn't this break the build? Always thought it has to have a certain structure.

Reply:
Does it?

Reply:
I know for distribution builds, we HAD to remove the certificate name. Perhaps it's OK for device (developer) builds too? Also, older Xcodes may have issue, but I'm not sure.

Reply:
It always used to be `iPhone Developer: XXXXXX` for device and `iPhone Distribution: XXXXXX` for dist-builds. If that's not required anymore: Cool! But we should research that before to confirm it actually does in all cases (Xcode 8+9 at least).

Reply:
Just confirmed on Xcode 9: It can be left blank, so `iPhone Developer` for device (aka `Debug`) and `iPhone Distribution` for dist-builds (aka `Release`).